ci/travis/linux/qt5/install.sh

```diff
@@ -1,15 +1,15 @@
 mkdir build
 cd build
 
-ln -s /usr/bin/ccache ${HOME}/osgeo4travis/bin/clang++-3.6
-ln -s /usr/bin/ccache ${HOME}/osgeo4travis/bin/clang-3.6
+ln -s ${HOME}/osgeo4travis/bin/ccache ${HOME}/osgeo4travis/bin/clang++-3.6
+ln -s ${HOME}/osgeo4travis/bin/ccache ${HOME}/osgeo4travis/bin/clang-3.6
 
 ccache -s
 
 export CXX="clang++-3.6"
 export CC="clang-3.6"
 export PATH=${HOME}/osgeo4travis/bin:${PATH}
-export PYTHONPATH=${HOME}/osgeo4travis/lib/python3/dist-packages/
+export PYTHONPATH=${HOME}/osgeo4travis/lib/python3.3/site-packages/
```

Comment:
Ah wait no, that's where the bundled python stuff (PyQt etc) from the pre-built binaries is shipped. So that's not registered as a python folder without an extra intervention.